src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.mockito.Mockito;
+
+public class FileWriterServiceImplTest {
+    private final FileWriterService fileWriterService = new FileWriterServiceImpl();
+
+    @Test
+    public void testWriteToFile_ok(@TempDir Path tempDir) throws IOException {
+        Path testFile = tempDir.resolve("testReport.csv");
+        String expectedContent = "fruit,quantity\napple,50\nbanana,20";
+
+        fileWriterService.writeToFile(testFile.toString(), expectedContent);
+
+        assertTrue(Files.exists(testFile), "File should exist after writing");
+        String actualContent = Files.readString(testFile);
+        assertEquals(expectedContent, actualContent, "File content should match expected content");
+    }
+
+    @Test
+    public void testWriteToFile_throwsExceptionOnInvalidPath_notOk() {
+        String invalidPath = "/invalid_directory/report.txt";
+        String reportContent = "Test report content";
+
+        assertThrows(RuntimeException.class, () -> {
+            fileWriterService.writeToFile(invalidPath, reportContent);
+        });
+    }
+
+    @Test
+    void testWriteToFile_shouldThrowRuntimeException_whenIoExceptionOccurs() throws IOException {
```

Comment:
You don't need this test, because you have already verified in your previous tests, that RuntimeException is thrown when an error occurs

Reply:
changed